backend/vendor/assets/javascripts/solidus_admin/select2_locales/select2_locale_sl-SI.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Select2 Slovenian translation
+ */
+(function ($) {
+    "use strict";
+
+    $.fn.select2.locales['de'] = {
```

Comment:
Yes, you are correct. I've fixed this in my local repo already, althouhg it had worked as commited too. On the other hand, it is also not enterily correct to name the slovenian translation of solidus sl-SI. It should be only sl (as iso code). Wrong name couses other problems, for example jquery validation on checkout pages... I can commit both fixes if desired, otherwise I'll update only current pull request.